routes/index.js

```diff
@@ -3,7 +3,7 @@ var router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+  res.sendFile('C:\\Users\\jpera\\WebstormProjects\\weatherapi\\views\\index.html');
```

Comment:
There should be a better way to handle this. A file name reference is very unreliable.

Reply:
Wasn't main focus of mr